examples/kotlin/src/main/kotlin/com/xebia/functional/xef/auto/Conversation.kt

```diff
@@ -0,0 +1,48 @@
+package com.xebia.functional.xef.auto
+
+import com.xebia.functional.xef.auto.llm.openai.getOrElse
+import com.xebia.functional.xef.auto.llm.openai.promptMessage
+
+suspend fun main() {
+    ai {
+
+        val email: String =
+            promptMessage(
+                """
+                |You are a Marketing Responsible and have the information about different products. You have to prepare 
+                |an email template with the personal information
+            """.trimMargin()
+            )
+        println("First question:\n $email")
```

Comment:
I would print first the prompt and then the answer in all three calls to the AI to show it like if it was a chat conversation